pychunkedgraph/app/segmentation/common.py

```diff
@@ -179,7 +179,9 @@ def handle_info(table_id):
 
     dataset_info = cg.dataset_info
     app_info = {"app": {"supported_api_versions": list(__api_versions__)}}
-    combined_info = {**dataset_info, **app_info}
+    hardcoded_info = {"n_bits_for_layer_id": cg._n_bits_for_layer_id,
+                      "cv_mip": cg._cv_mip, "n_layers": cg.n_layers}
```

Comment:
Avoid direct access to private attributes of the ChunkedGraph. Especially for `cg._cv_mip`, there is already a Getter available (`cg.cv_mip`). I am not sure about `cg._n_bits_for_layer_id`, but if there isn't yet, there should be one.